great_expectations/data_context/migrator/configuration_bundle.py

```diff
@@ -19,8 +19,6 @@
     DataContextVariables,  # noqa: TCH001
 )
 from great_expectations.data_context.types.base import (
-    CheckpointConfig,
```

Comment:
All this migrator code is to be deleted but ran into some cyclic import issues when trying to do so (cc @billdirks). Making minimal changes to get things working